### ISSUE TITLE
Station vars to control amount of cargo waiting on nearby industries

### ISSUE THREAD

**Issue by DorpsGek** [OpenTTD/OpenTTD#2309]
**_George_ opened the ticket and wrote:**

> R14263
> 
> Would it be possible to have station's animation according to the amount of cargo waiting on a nearby industry? (It's a feature request)
> The related 'object' for stations is town information, so stations have no access to industry information. It would be nice to have access to 'waiting cargo to be processed at the nearby industry' data; then we could animate stations accordingly. It wouldn't be feasible to have stockpile sprites with the station, as we don't know how much cargo is incoming; only outgoing cargo is supported.
> 
> Format: 6x var. Parameter - cargo number (in translation table)
> 
> Possible problems: there may be several industries that store this cargo in the station's area. Then the var should return the sum for all the industries.

**Reported version**: _trunk_
**Operating system**: _All_

---

**Comment by DorpsGek** [comment]
**_andythenorth_ closed the ticket.**

**Reason for closing: _Won't implement_**

> Show the cargo waiting with sprites on the industry - it's more direct, and more useful to the player.
> 
> Flyspray clean up, closing this one as it's > 5 years old, and it seems very unlikely to be provided.  Usual caveat applies: if you disagree, discuss with andythenorth in irc ;)  Thanks.

---
###### _This comment was imported from FlySpray: https://bugs.openttd.org/task/2309_
